flocker/testtools/_base.py

```diff
@@ -86,8 +93,25 @@ def __init__(self, *args, **kwargs):
 
     def setUp(self):
         super(AsyncTestCase, self).setUp()
+        # Need this to run _after_ the clean-up in CaptureTwistedLogs.
```

Comment:
I'm guessing that this comment is explaining why this cleanup is before the `useFixture`, since I'm assuming cleanups are run in reverse order to when they are added. If that is the case, I think you should make that explanation explicit.